dipy/segment/tests/test_feature.py

```diff
@@ -296,7 +322,14 @@ def infer_shape(self, streamline):
             return long(1)
 
         def extract(self, streamline):
-            return np.sum(np.sqrt(np.sum((streamline[1:] - streamline[:-1]) ** 2)))
+            return np.sum(
+                np.sqrt(
+                    np.sum(
+                        (streamline[
+                            1:] -
+                            streamline[
+                            :-
```

Comment:
Same comment as above. This is hard to read when it's broken into this many pieces.